plugins/catalog-backend-module-openapi-spec/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@backstage/plugin-catalog-backend-module-openapi-spec",
+  "version": "0.0.0",
```

Comment:
This should probably be at least `0.1.0` to allow future changes to align with semantic versioning